bundle/manifests/numaresources-operator.clusterserviceversion.yaml

```diff
@@ -599,4 +599,5 @@ spec:
   minKubeVersion: 1.23.0
   provider:
     name: Red Hat
+  replaces: numaresources-operator.v4.14.999-snapshot
```

Comment:
this is correct, but since we don't publish (yet?) on operatorhub.io, this is going to help us little. Let's have it nevertheless